config/sync/webform.webform.descriptive_metadata.yml

```diff
@@ -14,7 +14,298 @@ id: descriptive_metadata
+        sort:
+          field: title
+          direction: ASC
+  media:
+    '#type': webform_wizard_page
+    '#title': 'Attach Files'
+    '#open': true
+    '#prev_button_label': 'Back to Collections'
+    images:
+      '#type': webform_image_file
+      '#title': 'Upload Associated Images'
+      '#multiple': 10
+      '#description': 'This image will be used as a preview thumbnail.'
+      '#format': custom
+      '#format_html': '<img src="{{  item.link }}">'
+      '#format_items': ol
+      '#access_create_roles':
+        - authenticated
+      '#access_update_roles':
+        - authenticated
+      '#access_view_roles':
+        - authenticated
+      '#uri_scheme': s3
+      '#max_filesize': '256'
+      '#file_extensions': 'jp2 jpg jpeg png svg tiff'
+      '#file_name': '[random:hash:sha1]-[webform_submission:langcode]-image-[random:hash:sha1]'
+      '#sanitize': true
+    documents:
+      '#type': webform_document_file
+      '#title': 'Upload Associated Documents'
+      '#multiple': 1000
+      '#states':
+        visible:
+          - ':input[name="type"]':
+              value: DigitalDocument
+          - xor
+          - ':input[name="type"]':
+              value: Thesis
+          - xor
+          - ':input[name="type"]':
+              value: Book
+      '#file_placeholder': 'This text will appear in the preview of the panorama.'
+      '#file_name': '[random:hash:sha1]-[webform_submission:langcode]-doc-[random:hash:sha1]'
+      '#sanitize': true
+    audios:
+      '#type': webform_audio_file
+      '#title': 'Upload Audio File'
+      '#states':
+        visible:
+          ':input[name="type"]':
+            value: Podcast
+      '#states_clear': false
+      '#format_items': comma
+      '#file_preview': file
+      '#max_filesize': '2'
+      '#file_extensions': mp3
+    videos:
+      '#type': webform_video_file
+      '#title': Videos
+      '#states':
+        visible:
+          ':input[name="type"]':
+            value: Movie
+      '#states_clear': false
+      '#format_items': comma
+      '#file_preview': file
+      '#max_filesize': '50'
+      '#file_extensions': mp4
+    models:
+      '#type': managed_file
+      '#title': '3D Models'
+      '#states':
+        visible:
+          - ':input[name="type"]':
+              value: 3DModel
+          - or
+          - ':input[name="type"]':
+              value: Sculpture
+      '#states_clear': false
+      '#format_items': comma
+      '#max_filesize': '50'
+      '#file_extensions': stl
+    terms_of_service:
+      '#type': webform_terms_of_service
+      '#title': 'I agree to the {terms of service}'
+      '#required': true
+      '#required_error': 'You need to agree to the License Agreement'
+      '#terms_title': 'Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0) Licence'
+      '#terms_content': |
+        You agree on publishing your Panorama Image under&nbsp;<br />
```

Comment:
This needs to be, either removed (terms of service is useful but also quite specific to some use cases) or be super general. Seem like replacing Panorama, with Digital Object all over the place here could work (but don't follow that rule blindly)

Reply:
Due to the nature of a "Terms of Service" usually being applied to a specific project, I think section should be removed for a fresh Archipelago deployment.

Reply:
I totally agree. Wipe it!